translations/en.json

```diff
@@ -235,7 +235,6 @@
     "membership-title": "Membership requirement",
     "create-channel-title": "New channel",
     "edit-channel-title": "Edit channel",
-    "community-admins-will-review-your-request": "Community admins will review your request",
```

Comment:
It looks like this translation key is being used in e2e tests here: https://github.com/status-im/status-mobile/blob/9e5b8b5f9ccec332ba9d335688bfe99661572c7f/test/appium/tests/critical/test_deep_and_universal_links.py#L132.

I'm not familiar with this code. Should I do anything about this?
cc: @yevh-berdnyk / @status-im/mobile-qa

Reply:
Thank you!